### PR TITLE
Desktop: Linux: Remove dependency on lsb_release from install script

### LINKED ISSUE #5240: install script terminate with error on opensuse tumbleweed
installing joplin with the recommended script
`wget -O - https://raw.githubusercontent.com/laurent22/joplin/dev/Joplin_install_and_update.sh | bash`

end with error
```
Installing Joplin...
OK
Installing icon...
OK
bash: line 164: lsb_release: command not found
bash: line 165: lsb_release: command not found
```

## Environment

Joplin version: 2.1.9
Platform: opensuse tumbleweed vm (vmware player) on win10 host
seems that `lsb-release` is not a default package and cause the script to fail

NOTA : installed the `lsb-release` package manually 
return
```
mess-maker@localhost ~> lsb-release
LSB Version:    n/a
```
but the script terminate with no issue
```
Installing Joplin...
OK
Installing icon...
OK
Create Desktop icon...
OK
Joplin version 2.1.9 installed.
Cleaning up...
OK
```

## Steps to reproduce

1. install opensuse tumbleweed
2. launch the install script
3. 

## Describe what you expected to happen

install with no error

### PR DESCRIPTION
**Pull request by CalebJohn**
Fixes #5240 

The lsb_release module is used to detect certain platforms that need the `--no-sandbox` flag. This isn't needed for most platforms, and some platforms (notably openSUSE) don't have the lsb_release binary and don't need it.

### CODE REVIEW

File: Joplin_install_and_update.sh [hunk 1]
```diff
@@ -171,19 +171,23 @@ else
...
+# lsb_release isn't available on some platforms (e.g. opensuse)
+# The equivalent of lsb_release in OpenSuse is the file /usr/lib/os-release
+if command -v lsb_release &> /dev/null; then
+  DISTVER=$(lsb_release -is) && DISTVER=$DISTVER$(lsb_release -rs)
+  DISTCODENAME=$(lsb_release -cs)
+  #-----------------------------------------------------
+  # Check for "The SUID sandbox helper binary was found, but is not configured correctly" problem.
+  # It is present in Debian 10 Buster. A (temporary) patch will be applied at .desktop file
+  # Linux Mint 4 Debbie is based on Debian 10 and requires the same param handling.
+  if [ "$DISTVER" = "Debian10" ] || [ "$DISTVER" = "Linuxmint4" ] && [ "$DISTCODENAME" = "debbie" ]
```
Comment (tessus) [inline Joplin_install_and_update.sh:187]: I guess you can add `Debian11` to the list.

Comment (CalebJohn) [reply Joplin_install_and_update.sh]: Good call.

Comment (tessus) [reply Joplin_install_and_update.sh]: Thanks

**Comment by laurent22** [comment]
Looks good, thanks for the fix!

**Pull request merged**